plugins/mime/README.md

```diff
@@ -1,6 +1,6 @@
 ## JPEG XL MIME type
 
-If not already installed by the [Installing section of README.md](../../README.md#installing), then it can be done manually:
+If not already installed by the [Installing section of BUILDING.md](../../BUILDING.md#installing), then it can be done manually:
```

Comment:
As mentioned [here](https://github.com/libjxl/libjxl/issues/2183#issuecomment-1426981801), the whole mime folder should probably be dropped.

Reply:
Makes sense, but let's do that in a separate pull request.

Reply:
Sure